lib/commands/alert.js

```diff
@@ -19,6 +20,25 @@ commands.getAlertText = async function () {
   }
 };
 
+helpers.nativeSetAlertText = async function (text) {
+  let alert = await this.findNativeElementOrElements('class name', 'XCUIElementTypeAlert', true);
+  if (alert.length !== 1) {
+    throw new errors.NoAlertOpenError();
+  }
+  alert = util.unwrapElement(alert[0]);
+  let possibleTextFields = await this.findNativeElementOrElements('class name', 'XCUIElementTypeTextField', true, alert);
```

Comment:
I'd rather use single predicate search with query like `type == 'XCUIElementTypeTextField'  OR type == 'XCUIElementTypeSecureTextField'`

Reply:
or this can be even optimised further with class chain:
```**/XCUIElementTypeAlert/**/*[`type == 'XCUIElementTypeTextField' OR type == 'XCUIElementTypeSecureTextField'`]```

Reply:
I don't know enough about class chains to debug, and this one returns `Cannot parse class chain query`

Reply:
can you please attach the full trace?

Reply:
```
dbug JSONWP Proxy Proxying [POST /element/C9C22EF5-6D99-4F7F-8E80-5F5D43423687/elements] to [POST http://localhost:8100/session/46CA9370-32D5-49E5-ADB1-AAB6FD588A9D/element/C9C22EF5-6D99-4F7F-8E80-5F5D43423687/elements] with body: {"using":"class chain","value":"**/XCUIElementTypeAlert/**/*[`type == 'XCUIElementTypeTextField' OR type == 'XCUIElementTypeSecureTextField'`]"}
dbug JSONWP Proxy Got response with status 200: {"value":"Cannot parse class chain query '**/XCUIElementTypeAlert/**/*[`type == 'XCUIElementTypeTextField' OR type == 'XCUIElementTypeSecureTextField'`]'. Unexpected character detected at position 28:\n**/XCUIElementTypeAlert/**/* <----\n\n(\n\t0   CoreFoundation                      0x0000000108af41ab __exceptionPreprocess + 171\n\t1   libobjc.A.dylib                     0x0000000108189f41 objc_exception_throw + 48\n\t2   WebDriverAgentLib                   0x000000011de8f7f8 -[XCUIElement(FBClassChain) fb_descendantsMatchingClassChain:shouldReturnAfterFirstMatch:] + 376\n\t3   WebDriverAgentLib                   0x000000011deabce1 +[FBFindElementCommands elementsUsing:withValue:under:shouldReturnAfterFirstMatch:] + 897\n\t4   WebDriverAgentLib                   0x000000011deab710 +[FBFindElementCommands handleFindSubElements:] + 448\n\t5   WebDriverAgentLib                   0x000000011de970f3 -[FBRoute_TargetAction mountRequest:intoResponse:] + 179\n\t6   WebDriverAgentLib                   0x000000011d...
```
So...
```
Cannot parse class chain query '**/XCUIElementTypeAlert/**/*[`type == 'XCUIElementTypeTextField' OR type == 'XCUIElementTypeSecureTextField'`]'. Unexpected character detected at position 28:\n**/XCUIElementTypeAlert/**/* <----

(
    0   CoreFoundation                      0x0000000108af41ab __exceptionPreprocess + 171
    1   libobjc.A.dylib                     0x0000000108189f41 objc_exception_throw + 48
    2   WebDriverAgentLib                   0x000000011de8f7f8 -[XCUIElement(FBClassChain) fb_descendantsMatchingClassChain:shouldReturnAfterFirstMatch:] + 376
    3   WebDriverAgentLib                   0x000000011deabce1 +[FBFindElementCommands elementsUsing:withValue:under:shouldReturnAfterFirstMatch:] + 897
    4   WebDriverAgentLib                   0x000000011deab710 +[FBFindElementCommands handleFindSubElements:] + 448\n\t5   WebDriverAgentLib                   0x000000011de970f3 -[FBRoute_TargetAction mountRequest:intoResponse:] + 179
    6   WebDriverAgentLib                   0x000000011d...
```

Reply:
Please do ```**/XCUIElementTypeAlert/**/XCUIElementTypeAny[`type == 'XCUIElementTypeTextField' OR type == 'XCUIElementTypeSecureTextField'`]``` instead